apps/deploy-web/src/utils/stripeUtils.ts

```diff
@@ -0,0 +1,12 @@
+import { loadStripe, type Stripe } from "@stripe/stripe-js";
+
+import { browserEnvConfig } from "@src/config/browser-env.config";
+
+export function getStripe(): Promise<Stripe | null> {
```

Comment:
suggestion(non-blocking): would be nice to move this into some `StripeService` or `PaymentService`

Reply:
I create a service for it